### ISSUE TITLE
See tasks after CTF is over

### ISSUE THREAD

**Issue by arturo182** [balidani/tinyctf-platform#9]
Would be nice to turn the tasks into read-only mode once the CTF is over so teams can still see the challenge description, link and how many teams solved it. 
It'd really help when writing writeups :smiley:

**Comment by arturo182** [comment]
I based this on https://ctf.internetwache.org but I can't find the code responsible for that, so it might be their modification.

**Comment by gehaxelt** [comment]
@arturo182 Yes, this is indeed our modification. We'll create a fork, push our modifications and create a PR eventually.

**Comment by arturo182** [comment]
@gehaxelt you the man!
